src/app/auth/auth.component.ts

```diff
@@ -50,13 +50,13 @@ export class AuthComponent implements OnInit, OnDestroy {
    * Informs user of missing profiles file.
    * @param profilesDetails - profiles file information.
    */
-  onProfilesMissing(profilesDetails: {}): void {
-    this.profileLocationPrompt = profilesDetails['visible']
+  onProfilesMissing(profilesDetails: {visible: boolean, fileName: string, fileDirectory: string}): void {
```

Comment:
`visible` -> `isFileVisible`

Reply:
The visibility is for the message that is displayed at the bottom of the login form. Shall I change it to `isMessageVisible` ?